component/core/src/main/java/org/exoplatform/social/core/model/BannerAttachment.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
We don't have Banner attachment in eXo Platform 4.4, why backporting adding this ?